spec/requests/media_auth_request_spec.rb

```diff
@@ -2,17 +2,11 @@
 
 RSpec.describe "Authentication for Media requests", type: :request do
 
-  let(:allowed_loc) { 'ip.address1' }
   let(:user_no_loc_no_webauth) { User.new }
-  let(:user_no_loc_webauth) { User.new(webauth_user: true) }
```

Comment:
The changes here confuse me a bit.  Do we no longer care about:
* various user scenarios when an object is both location and stanford restricted. 
* the user scenario when user is webauthed but not stanford.

Or, is that not testable here any more, but is tested somewhere else?   Because we certainly will still have the possibility of all these types of users interacting with stanford-only, with location-only and with both for streaming ... and I assume that is still handled by stacks?